description: 'net: tulip: de4x5: fix the problem that the array ''lp->phy[8]'' may
  be out of bound'
references:
- https://www.cve.org/CVERecord?id=CVE-2021-47547
- https://git.kernel.org/stable/c/ec5bd0aef1cec96830d0c7e06d3597d9e786cc98
- https://git.kernel.org/stable/c/142ead3dc70411bd5977e8c47a6d8bf22287b3f8
- https://git.kernel.org/stable/c/d3dedaa5a601107cfedda087209772c76e364d58
- https://git.kernel.org/stable/c/2c1a6a9a011d622a7c61324a97a49801ba425eff
- https://git.kernel.org/stable/c/77ff166909458646e66450e42909e0adacc99049
- https://git.kernel.org/stable/c/f059fa40f0fcc6bc7a12e0f2a2504e9a4ff74f1f
- https://git.kernel.org/stable/c/12f907cb11576b8cd0b1d95a16d1f10ed5bb7237
- https://git.kernel.org/stable/c/61217be886b5f7402843677e4be7e7e83de9cb41
- https://lore.kernel.org/linux-cve-announce/2024052441-CVE-2021-47547-f3e7@gregkh/
- https://git.kernel.org/linus/61217be886b5f7402843677e4be7e7e83de9cb41
comments:
  cip/cip-kernel-sec: There is no mention of the commit that introduced the bug in
    the CVE announcement.Fixed in v5.16-rc2.
  debian/carnil: First introducing commit could not be determined.
fixed-by:
  cip/4.19: [2c1a6a9a011d622a7c61324a97a49801ba425eff]
  cip/4.19-rt: [2c1a6a9a011d622a7c61324a97a49801ba425eff]
  cip/4.4: [ec5bd0aef1cec96830d0c7e06d3597d9e786cc98]
  cip/4.4-rt: [ec5bd0aef1cec96830d0c7e06d3597d9e786cc98]
  cip/4.4-st: [ec5bd0aef1cec96830d0c7e06d3597d9e786cc98]
  cip/5.10: [f059fa40f0fcc6bc7a12e0f2a2504e9a4ff74f1f]
  cip/5.10-rt: [f059fa40f0fcc6bc7a12e0f2a2504e9a4ff74f1f]
  debian/buster-security: ['version:debian/4.19.232-1']
  mainline: [61217be886b5f7402843677e4be7e7e83de9cb41]
  stable/4.19: [2c1a6a9a011d622a7c61324a97a49801ba425eff]
  stable/4.4: [ec5bd0aef1cec96830d0c7e06d3597d9e786cc98]
  stable/5.10: [f059fa40f0fcc6bc7a12e0f2a2504e9a4ff74f1f]
  stable/5.15: [12f907cb11576b8cd0b1d95a16d1f10ed5bb7237]
  stable/5.4: [77ff166909458646e66450e42909e0adacc99049]
  ubuntu/bionic: [49872584b08c18fbfa79eff251a0aef4c895493d]
  ubuntu/focal: [79ab9e405710397c200c2c3b779a1ad792b6d4b3]
